src/api/api.js

```diff
@@ -0,0 +1,51 @@
+export async function getUserInfo() {
```

Comment:
Info라는 표현없이도 의미를 드러내기에 충분할 것 같은데 어떠신가요~?
```suggestion
export async function getUser() {
```